docs/networking/harvester-network.md

```diff
@@ -41,30 +38,34 @@ Below is a use case of the VLAN in Harvester.
 
 ## Enabling VLAN in the Harvester UI
 
-Enable VLAN via going to **Setting > vlan** to enable VLAN and input a valid default physical NIC name for the VLAN.
+Enable VLAN via going to **Setting > vlan** to enable VLAN and input a valid default physical NIC name for the VLAN. 
 
-It is recommended to choose a separate NIC for the VLAN other than the one used for the management network (the one selected during the Harvester installation) for better network performance and isolation.
+It is recommended to choose a separate NIC for the VLAN other than the one used for the management network (the one selected during the Harvester installation) for better network performance and isolation. 
 
 !!! note
-Modifying the default VLAN network setting will not change the existing configured host networks.
+    Modifying the default VLAN network setting will not change the existing configured host networks.
+    
+!!! note
+    Harvester support bond interface but only can be created during PEX Boot installation. Here is [an example](/install/harvester-configuration/#example_11).
```

Comment:
```suggestion
    Harvester supports configuring bond interfaces but it can only be created during PEX Boot installation. Here is [an example](/install/harvester-configuration/#example_11).
```